src/client/components/Story/Body.js

```diff
@@ -109,13 +111,20 @@ Body.propTypes = {
   jsonMetadata: PropTypes.string,
   full: PropTypes.bool,
   rewriteLinks: PropTypes.bool,
+  exitPageSetting: PropTypes.bool,
 };
 
 Body.defaultProps = {
   body: '',
   jsonMetadata: '',
   full: false,
   rewriteLinks: false,
+  exitPageSetting: true,
 };
 
-export default Body;
+export default connect(
```

Comment:
I would rather keep Body as dumb component and pass this prop instead (or create BodyContainer).

Reply:
I think a BodyContainer would be best otherwise we would modify EmbeddedCommentForm, CommentForm for instance and connect them with the settings which doesn't make sense.
What is your final word ?

Reply:
I prefer having small container components that just wrap dumb component, so I think BodyContainer would work the best.